test/fixtures/android-appium-test/features/steps/steps.rb

```diff
@@ -0,0 +1,11 @@
+When('I issue a command to notify a {string} handled error') do |metadata|
```

Comment:
Are we intending to bake these steps into Maze-runner with a pre-defined structure, or are we happy for them to be implemented on a per-platform basis for now?

Reply:
My original implementation included defined Command classes for RunScenario and StartBugsnag.  I personally liked the abstraction and it saved a couple of lines of code, but Nick didn't see the point and was concerned about having to wait for changes to Maze Runner before being able to implement new commands.  I can see it both ways, really, putting more into Maze Runner can save boilerplate code, but can also put the onus on it to do more testing and/or be more reactive to changes that are needed.